### PR TITLE
feature: I've added a new feature!

### PR DESCRIPTION
**Pull request by csmart**
Hi Ben please merge this

It is awesome

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # chenbris
 Watch closely. Magic.
+Lots of magic, special magic.
```
Comment (benhegarty) [inline README.md]: Dude. Really?

Comment (csmart) [inline README.md]: Ahh, yeah..

Comment (benhegarty) [inline README.md]: Okay then...

**Comment by benhegarty** [comment]
Thanks bruv

**Comment by csmart** [comment]
Sweet!

**Pull request merged**